cypress/e2e/common/candidate-common.ts

```diff
@@ -0,0 +1,33 @@
+import { Given, When } from "@badeball/cypress-cucumber-preprocessor";
+import CandidatePayloadInit from "../../support/Initializers/Pyload Initializers/candidate-payload-Init";
+import { RecruitmentApiHelper } from "../../support/Helpers/Recruitment Page Helpers/api-helper";
+import { CandidatePage } from "../../support/POM/Recruitment Page Files/candidates-page";
+
+Given("Admin added a new candidate", () => {
```

Comment:
you can declare the common steps  as:  `Given("Common Step: Admin added a new candidate",() => {.....}`
and call it in every scenario you want as Common Step: Admin added a new candidate